[TD-29] infographic wizard

emullaney opened the pull request:
New infographic wizard on the Panel, Cohort, Poll and Fetch Run Pages

Follows UX mockup from https://caktus.atlassian.net/browse/TD-29 but the order has been changed to the above order (because you need contacts before you can track polls). The Jira ticket has been updated as well.

dpoirier commented on tracpro/templates/workflow_tracker_steps.html:
```diff
@@ -0,0 +1,40 @@
+    </a>
+    <div class="col-md-2 hidden-sm hidden-xs workflow-spacer"></div>
+
+    <a href="{% url 'orgs_ext.org_fetchruns' %}">
```
We should only display the 4th circle if the user is authorized to do fetchruns.

(```  {% if request.user.is_superuser or request.user.is_staff %}``` is what we've used elsewhere.)

emullaney replied:
Actually that should be true of the entire wizard! I will wrap it.

emullaney replied:
I believe that staff and super users should be able to get to any of these pages so that's why I didn't include it to begin with.

dpoirier commented on tracpro/static/less/trac.less:
```diff
@@ -203,4 +203,44 @@ footer {
 /* Override datepicker colors */
 .ui-datepicker select.ui-datepicker-month, .ui-datepicker select.ui-datepicker-year {
   color: #000000;
```
When I narrow my window, the circles and text end up scrunched on top of each other while there's still horizontal space that they could spread out in.

![image](https://cloud.githubusercontent.com/assets/2788679/26582128/e54614f2-450d-11e7-8e0c-ef5e9b7d5e3a.png)

emullaney replied:
I'm fixing the scrunching by hiding the words on small screen sizes. Fix coming.

dpoirier commented on tracpro/static/less/trac.less:
```diff
@@ -203,4 +203,44 @@ footer {
   color: #000000;
-}
\ No newline at end of file
+}
```
I noticed some differences from the mockup:

* On the mockup, the circles had a very light circle drawn around them just a few pixels outside the colored circle.
* The text in the unselected circles was dark rather than light.
* The labels were bold.
* The circles appear smaller in the mockup, though I haven't measured them.

emullaney replied:
I've made a few improvements here. I think it's close enough to the mockup now. Let me know!

emullaney commented:
I have
* Updated the styles for the workflow tracker
* Added to the workflow tracker to My Organization
* Fixed the drop down order in the navigation to match the new tracker

Ready for review again @dpoirier

dpoirier commented:
Things look okay to me, just a little different stylistically from the mockup. Your call if you want to :shipit:.
